index.js

```diff
@@ -37,6 +36,7 @@ function Ciper(options) {
   //
   this.admins = options.admins || [];
   this.nodeType = options.nodeType || '';
+  this.type = options.type || 'pr';
```

Comment:
we should add validation here after this. `if(validTypes.indexOf(this.type) === -1) throw new Error(`Invalid type ${this.type}`)